src/schema/v2/homeView/sections/ShowsForYou.ts

```diff
@@ -10,4 +17,24 @@ export const ShowsForYou: HomeViewSection = {
     title: "Shows for You",
   },
   requiresAuthentication: true,
+
+  shouldBeDisplayed: (_context: ResolverContext) => true,
+
+  resolver: withHomeViewTimeout(async (parent, args, context, info) => {
```

Comment:
Is defining the resolver for a section's (main) connection in the parent (the section type) the preferred pattern?

Reply:
The _section type_ (HomeViewSectionShows) defines the interface and the _section instance_ (ShowsForYou) defines the behavior/resolution.

This is the general pattern that allows different instances of the HomeViewSectionShows section type to specify different loading behavior but in the end still produce a ShowsConnection to clients.

Reply:
That makes sense! Thanks for the explanation 🙏